Yank/commands/script.py

```diff
@@ -25,14 +25,17 @@
 YANK script
 
 Usage:
-  yank script (-y=FILEPATH | --yaml=FILEPATH)
+  yank script (-y=FILEPATH | --yaml=FILEPATH) [--platform=PLATFORM]
```

Comment:
precision should be specified in the YAML file since we may want to regress over precision choices.